app/models/tag.rb

```diff
@@ -0,0 +1,7 @@
+class Tag < ApplicationRecord
+
+  belongs_to :organization
+
+  validates :name, :organization, presence: true
+
```

Comment:
That's unclear, but I think we'd kind of agreed that the IR would keep what members have directly contributed.  We should ask Dante what his thoughts are though since we could just send them on to linkify and not need to retain them